### DISCUSSION TITLE
Limiting Results with Full Text Search

### DISCUSSION THREAD

**Discussion by nbibler** [prisma/prisma#24263] [Q&A]
### Question

When using the `fullTextSearch` feature introduced v3.6.0, is it possible to limit the results to a minimum relevance value?

For example, the following query will return _all records_ in the table, in a descending relevance score order, regardless of their full text search relevance score:

```javascript
prismaClient.tableName.findMany({
  _relevance: {
    fields: ["field1", "field2", "field3"],
    search: "my & term",
    sort: "desc",
  },
})
```

Is it possible through the `fullTextSearch` feature to limit the results to only those results with a relevance value above some arbitrary value? For example:

```javascript
prismaClient.tableName.findMany({
  _relevance: {
    fields: ["field1", "field2", "field3"],
    search: "my & term",
    sort: "desc",
  },
  where: { _relevance: { gt: 0.4 } }
})
```

**Comment by ludralph** [comment]
Hi @nbibler 👋 

Thank you for raising this question.

> Is it possible through the fullTextSearch feature to limit the results to only those results with a relevance value above some arbitrary value? For example:

This is currently not possible in the current implementation of FTS. We do have a [feature request](https://github.com/prisma/prisma/issues/22390) that is similar to what you are asking for. You can please add your usecase to the open issue and leave a 👍 to help our engineers prioritize the issue.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
